demisto_sdk/commands/format/tests/test_formatting_generic_test.py

```diff
@@ -364,9 +378,11 @@ def test_adds_period_to_description(
         the description in the YAML data should have a period added if is not end with url.
     """
     yml_data = {
```

Comment:
is this a fusion integration/script yml? If so, please split the test - once for script, once for integration. If not, please explain.

Reply:
Done